src/draw.typ

```diff
@@ -306,6 +306,30 @@
   ),)
 }
 
+/// Draw body on z-index layer
+///
+/// This can be used to draw elements behind already drawn elements by
+/// using a lower layer value (i.e -1).
+/// 
+/// - layer (integer, float): Z-Index to draw the children at
```

Comment:
why a float?

Reply:
Why not?

Reply:
To me layers should be discrete not continuous but I guess it doesn't really effect much.